src/bindings/python/tests/test_runtime/test_core.py

```diff
@@ -429,3 +429,72 @@ def test_infer_new_request_return_type(device):
     assert arr.shape == (10,)
     assert arr.dtype == "float32"
     assert arr.nbytes == 40
+
+
+def test_read_model_node_names_with_spaces():
```

Comment:
I think we can remove this test because we cover all changes on C++ level :)

Reply:
Test removed :)